docs/root/intro/arch_overview/upstream/load_balancing/slow_start.rst

```diff
@@ -0,0 +1,60 @@
+.. _arch_overview_load_balancing_slow_start:
+
+Slow start mode
+===============
+
+Slow start mode is a configuration setting in Envoy to progressively increase amount of traffic for newly added upstream endpoints.
+With no slow start enabled Envoy would send a proportional amount of traffic to new upstream endpoints.
+This could be undesirable for services that require warm up time to serve full production load and could result in request timeouts, loss of data and deteriorated user experience.
+
+Slow start mode is a mechanism that affects load balancing weight of upstream endpoints and can be configured per upstream cluster.
+Currently, slow start is supported in Round Robin and Least Request load balancer types.
+
+Users can specify a :ref:`slow start window parameter<envoy_v3_api_field_config.cluster.v3.Cluster.SlowStartConfig.slow_start_window>` (in seconds), so that if endpoint "cluster membership duration" (amount of time since it has joined the cluster) is within the configured window, it enters slow start mode.
+During slow start window, load balancing weight of a particular endpoint will be scaled with time factor, e.g.:
+
+.. math::
+
+  NewWeight = {Weight*TimeFactor}^\frac{1}{Aggression}
+
+where,
+
+.. math::
+
+  TimeFactor = \frac{max(TimeSinceStartInSeconds,1)}{SlowStartWindowInSeconds}
+
+As time progresses, more and more traffic would be sent to endpoint within slow start window.
+
+:ref:`Aggression parameter<envoy_v3_api_field_config.cluster.v3.Cluster.SlowStartConfig.aggression>` non-linearly affects endpoint weight and represents the speed of ramp-up.
+By tuning aggression parameter, one could achieve polynomial or exponential speed for traffic increase.
+Below simulation demonstrates how various values for aggression affect traffic ramp-up:
+
+.. image:: /_static/slow_start_aggression.svg
+   :width: 60%
+   :align: center
+
+Whenever a slow start window duration elapses, upstream endpoint exits slow start mode and gets regular amount of traffic according to load balancing algorithm.
+Its load balancing weight will no longer be scaled with runtime bias and aggression. Endpoint could also exit slow start mode in case it leaves the cluster.
+
+To reiterate, endpoint enters slow start mode:
+  * If no active healthcheck is configured per cluster, immediately if its cluster membership duration is within slow start window.
+  * In case an active healthcheck is configured per cluster, when its cluster membership duration is within slow start window and endpoint has passed an active healthcheck.
```

Comment:
Sorry not sure where we landed on this, but wouldn't it make sense in this case to start slow start once the first active health check has passed? cc @tonya11en 

I would be fine if this were a future enhancement but just curious if we discussed this previously.

Reply:
That does make a lot more sense.. it just never occurred to me when I looked at it.

I'm in favor of starting the window upon the first successful HC, but given how long this PR has dragged on I think it should be in a subsequent diff. If we're in agreement, then I think a simple issue and link to this comment thread should suffice to track it.

Reply:
Yes agree let's land and iterate. An issue is fine.

Reply:
+1 on opening a new issue (will do it tomorrow). also feeling a little bit exhausted from this pr taking so long

Reply:
This issue still needs to be opened please or if it was link it here?

Reply:
Issue link: https://github.com/envoyproxy/envoy/issues/18351